setup.py

```diff
@@ -22,27 +20,32 @@
       long_description=open(README).read(),
       author='Common workflow language working group',
       author_email='common-workflow-language@googlegroups.com',
-      url="https://github.com/common-workflow-language/common-workflow-language",
-      download_url="https://github.com/common-workflow-language/common-workflow-language",
+      url="https://github.com/common-workflow-language/"
+      "common-workflow-language",
+      download_url="https://github.com/common-workflow-language/"
+      "common-workflow-language",
+      extras_require={':python_version>="2.7"': ['typing']},
       license='Apache 2.0',
       packages=["cwltool"],
-      package_data={'cwltool': ['schemas/draft-3/*.yml',
-                                'schemas/draft-3/*.md',
-                                'schemas/draft-3/salad/schema_salad/metaschema/*.yml',
-                                'schemas/draft-3/salad/schema_salad/metaschema/*.md']},
+      package_data={'cwltool': [
+          'schemas/draft-3/*.yml',
+          'schemas/draft-3/*.md',
+          'schemas/draft-3/salad/schema_salad/metaschema/*.yml',
+          'schemas/draft-3/salad/schema_salad/metaschema/*.md']},
```

Comment:
Agree.  Either the schema salad spec should go in its own repo or I should use subtree split so that the cwl spec only imports the metaschema and not all the code.

On March 11, 2016 4:51:05 AM EST, "Michael R. Crusoe" <notifications@github.com> wrote:
>>        license='Apache 2.0',
>>        packages=["cwltool"],
>> -      package_data={'cwltool': ['schemas/draft-3/*.yml',
>> -                                'schemas/draft-3/*.md',
>> -                               
>'schemas/draft-3/salad/schema_salad/metaschema/*.yml',
>> -                               
>'schemas/draft-3/salad/schema_salad/metaschema/*.md']},
>> +      package_data={'cwltool': [
>> +          'schemas/draft-3/*.yml',
>> +          'schemas/draft-3/*.md',
>> +          'schemas/draft-3/salad/schema_salad/metaschema/*.yml',
>> +          'schemas/draft-3/salad/schema_salad/metaschema/*.md']},
>
>It would be great if we didn't both depend on schema_salad and embed it
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/common-workflow-language/cwltool/pull/50/files#r55809651

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.